src/ceph.in

```diff
@@ -853,7 +906,21 @@ def main():
         return 1
 
     if parsed_args.help:
-        return do_extended_help(parser, childargs)
+        return do_extended_help(parser, childargs, ('mon', ''), ' '.join(childargs))
+
+    # implement "tell service.id help"
+    if len(childargs) >= 3 and childargs[0] == 'tell' and childargs[2] == 'help':
+        if validate_target(childargs[1]):
+            target = childargs[1].split('.')
```

Comment:
please move `target = childargs[1].split('.')` out of the `if` block, and pass `target` to `validate_target()`, this would simplify the code a little bit.